src/components/Panel/PanelHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react';
+
+import { PanelHeaderProps } from './Panel.types';
+
+import { SystemUIButton } from '../Button';
+import { IconName } from '../Icon';
+
+import styles from './panel.module.scss';
+
+export const PanelHeader: FC<PanelHeaderProps> = ({
+    onClose,
+    closeIcon = IconName.mdiClose,
+    title,
+    actionDefaultButtonProps,
+    actionButtonOneProps,
+    actionButtonTwoProps,
+}) => {
+    return (
+        <div className={styles.logoGradientHeaderWrapper}>
+            <div className={styles.headerTitle}>
+                {!!actionDefaultButtonProps && (
+                    <SystemUIButton {...actionDefaultButtonProps} />
+                )}
+                {title}
+            </div>
+            <div className={styles.headerActionButtons}>
+                {!!actionButtonOneProps && (
```

Comment:
```suggestion
                {actionButtonOneProps && (
```
I know you are used to this pattern cause of react native, but no need here